src/utils/TriqlerConverter.cpp

```diff
@@ -132,7 +131,7 @@ class TOPPTriqlerConverter final :
         ExperimentalDesign::SampleSection sampleSection = design.getSampleSection();
 
         ConsensusMap consensus_map;
-        ConsensusXMLFile().load(arg_in, consensus_map);
+        FileHandler().loadConsensusFeatures(arg_in, consensus_map, {FileTypes::CONSENSUSXML});
```

Comment:
here you specify the type and I think in others you don't
because here arg_in could be everything? 
what is the semantics without?

Reply:
The only valid input type here is `setValidFormats_(param_in, ListUtils::create<String>("consensusXML"), true);` So I set it to that. This is to be conservative and ensure that we don't get any regressions with the major new behavior. The FileHandler invocations that don't have any limiting types all existed prior to this refactor and so I didn't touch them.

Reply:
I think it is fine. Just making sure I understand how the semantics are.